server/src/main/java/org/opensearch/index/shard/RemoteStoreRefreshListener.java

```diff
@@ -262,6 +262,7 @@ public void onFailure(Exception e) {
         updateFinalStatusInSegmentTracker(successful.get(), bytesBeforeUpload, startTimeInNS);
         // If there are failures in uploading segments, then we should retry as search idle can lead to
         // refresh not occurring until write happens.
+        logger.debug("syncSegments runStatus={}", successful.get());
```

Comment:
In `onSuccessfulSegmentsSync`, we set the min seq number to keep. If it is not logged in translog, we may need to log that as well.

Reply:
Done